src/test/java/GmailClientTest.java

```diff
@@ -131,15 +139,45 @@ public void getQueryStringDontFilterForImportantTag() {
   }
 
   @Test
-  public void getQueryStringCombined() {
-    // Should return query matching all specified rules
-    // trailing spaces on sub-queries do not matter for this method, so they will not be considered
-    String multipleFilterQuery =
-        GmailClient.emailQueryString(ONE_UNIT_OF_TIME, DAYS_UNIT, true, true, SAMPLE_EMAIL);
-
-    Assert.assertTrue(multipleFilterQuery.contains(ONE_DAY_QUERY));
-    Assert.assertTrue(multipleFilterQuery.contains(UNREAD_EMAILS_QUERY));
-    Assert.assertTrue(multipleFilterQuery.contains(IMPORTANT_EMAILS_QUERY));
-    Assert.assertTrue(multipleFilterQuery.contains(FROM_EMAIL_QUERY));
+  public void getQueryStringNoSubjectWords() {
+    String subjectLineQuery = GmailClient.wordsInSubjectLineQuery(EMPTY_SUBJECT_LINE_WORDS);
+
```

Comment:
very minor nit: this two-line test has a newline in between and the two-line test `combineQueriesNonePresent` doesn't, would suggest either removing this newline or adding a newline after line 168

Reply:
Consider it done!